src/NosCore.GameObject/Character.cs

```diff
@@ -77,7 +80,7 @@ public Character()
 
         public bool IsFriendListFull
         {
-            get => CharacterRelations.Where(s => s.Value.RelationType == CharacterRelationType.Friend).ToList().Count
```

Comment:
Why is it returning null values ? I would préfère fixing it instead of using count

Reply:
still todo